src/utils/collection.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Removes duplicate records from a list, given a field to compare
+ * against. This is used to control for duplicate entries returned
+ * from API endpoints.
+ *
+ * Ultimately, we can't control this. However it is important to
+ * deduplicate search results for React indexing and user experience.
```

Comment:
This looks good to me, but I also feel like it should be handled upstream whenever possible. What do you think about passing a warning message along whenever appropriate?

Reply:
Great idea, 👍